tests/test_modified_utf8.py

```diff
@@ -20,3 +22,24 @@ def test_decode_modified_utf8():
         # We simply need to make sure no encoding exceptions are raised
         # when we parse the ClassFile.
         ClassFile(fin)
+
+
+def test_encode_utf8_1():
+    # string containing byte 00
+    str1 = u'1\x002'
+    bb = encode_modified_utf8(str1)
+    assert '31c08032' == binascii.hexlify(bb), binascii.hexlify(bb)
+
+    # Unicode supplementary character U+10400
+    str1 = u'\uD801\uDC00'
+    bb = encode_modified_utf8(str1)
+    assert 'eda081edb080' == binascii.hexlify(bb), binascii.hexlify(bb)
+
+def test_decode_utf8_1():
```

Comment:
Would you be able to expand the docstrings on these tests? Happy to merge after that.